src/ui/components/CouncilSummaryTeaser.tsx

```diff
@@ -5,17 +5,17 @@ import {Padder} from '@ui/components/Padder';
 import {SectionTeaserContainer} from '@ui/components/SectionTeaserContainer';
 import ProgressChartWidget from '@ui/components/ProgressWidget';
 import StatInfoBlock from '@ui/components/StatInfoBlock';
-import {useCouncilSummary} from 'src/api/hooks/useCouncilSummary';
 import {LoadingBox} from '@ui/components/LoadingBox';
 import {Card} from '@ui/library';
 import {AccountTeaser} from '@ui/components/Account/AccountTeaser';
+import {useCouncil} from 'src/api/hooks/useCouncil';
 
 type PropTypes = {
   onPress: () => void;
 };
 
 export function CouncilSummaryTeaser(props: PropTypes) {
-  const {data, loading} = useCouncilSummary();
+  const {data, loading} = useCouncil();
```

Comment:
This will be good for caching the council data before navigating to the screen but If we use this query here the app will always fetch everything (members, runners up, etc) even if the user never goes to the council screen.

Reply:
Yes, we can have a separate summary query which only fetches limited information.